storage/ops.go

```diff
@@ -1318,21 +1405,147 @@ func FindModifyInstallTargets(medias []*BlockDevice) []InstallTarget {
 	return sortInstallTargets(installTargets)
 }
 
+// FindAdvancedInstallTargets creates a list of advanced installation targets
+// We use Partition Labels to tag and convey which partitions should be used
+// for an advanced installations.
+//	CLR_BOOT = The /boot partition; must be vfat
+//	CLR_SWAP = A swap partition to use; can be more than one
+//	CLR_ROOT = The / root partition; must be ext[234] or xfs
+//		due to clr-boot-manager
+//	CLR_MNT = Any additional partitions that should be
+//		included in the install like /srv, /home, ...
```

Comment:
I'm having trouble mounting on /home

* Gparted: 
``` /dev/sda4, CLR_MNT_/home,  ext4```
* Confirm Installation scroll entry:
```sda4: Use existing partition. [/home]```

After installing, when I boot into my image, /dev/sda4 exists, but it is not mounted.